spec/factories/service_providers.rb

```diff
@@ -1,7 +1,7 @@
 FactoryGirl.define do
   factory :service_provider do
     sequence(:friendly_name) { |n| "test-service_provider-#{n}" }
-    sequence(:issuer) { |n| "test-service_provider-#{n}" }
+    sequence(:issuer) { |n| "urn:gov:gsa:SAML:2.0.profiles:sp:sso:DEPT:APP-#{n}" }
```

Comment:
Hmm, new issuers will need to change anyway. Perhaps a message saying something to the end of `Issuer is invalid and needs to be updated` in the UI if the department and agency strings can't be parsed out of the issuer string?